### ISSUE TITLE
Request for hyperparameters 

### ISSUE THREAD

**Issue by Tlhey** [nigelnnk/FairGAD#2]
I am currently working with your dataset and would like to use it as a baseline in my research. To ensure that I can accurately replicate your experiments and achieve comparable results, could you kindly provide the hyperparameters used for different pygod models that were used during your study?

**Comment by nigelnnk** [comment]
Hi yuchen, could you please refer to `fairGAD/test_fair_fitting.py`? The hyperparameters used are the  default ones listed in the file. Were you looking for any specific hyperparameters?

**Comment by Tlhey** [comment]
Hi, I was wondering if you could provide the hyperparameter of vanilla models results of CoLA, DOMINANT, CONAD, and DONE (without fair regularization method like EDITS and Fairwalk)?

**Comment by nigelnnk** [comment]
I see, for running models, the general parameters for all models are found in lines 31-60 of `fairGAD/test_fair_fitting.py` (e.g. batch size is 16384 and the seed is 1). For the other parameters specific to each model, we used the default parameters as stated in the model class files found in `pygod-v0.3/pygod/models` (e.g. for DOMINANT, parameters like dropout and weight decay are found in `pygod-v0.3/pygod/models/dominant.py`).

**Comment by Tlhey** [comment]
Thank you!
